core/web/webauthn_controller.go

```diff
@@ -27,22 +29,23 @@ func NewWebAuthnController(app chainlink.Application) WebAuthnController {
 }
 
 func (c *WebAuthnController) BeginRegistration(ctx *gin.Context) {
-	orm := c.App.SessionORM()
-	user, err := orm.FindUser()
-	if err != nil {
-		jsonAPIError(ctx, http.StatusInternalServerError, fmt.Errorf("failed to obtain current user record: %+v", err))
+	user, ok := auth.GetAuthenticatedUser(ctx)
+	if !ok {
+		jsonAPIError(ctx, http.StatusInternalServerError, pkgerrors.Errorf("failed to obtain current user from context"))
```

Comment:
Are these omitting the formerly wrapped errors intentionallly? If so, is the `*f(` still correct?

Reply:
In this case we don't get an error back from the above call, so just the opaque message here